app/serializers/tool_group_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ToolGroupSerializer < ActiveModel::Serializer
+  attributes :id, :name, :suggestions_weight
```

Comment:
the `_` in `suggestions_weight` isn't translating properly to "suggestions-weight" for the API endpoints, what needs to happen to fix it is to specify an explicit key for that attribute. see: https://github.com/CruGlobal/mobile-content-api/blob/master/app/serializers/resource_serializer.rb#L6

```suggestion
  attributes :id, :name
  attribute :suggestions_weight, key: "suggestions-weight"
```

Reply:
@frett the change is there